practice-two/src/javascript/modules/filterNotes.js

```diff
@@ -0,0 +1,34 @@
+import httpUtils from '../utils/httpUtils';
+import { renderNote, renderAllNotes } from './getNotes';
+/* global document */
+
+async function renderNotesByCategory(category) {
+  try {
+    const allNotes = await httpUtils.getNotes();
```

Comment:
@quanghuy1799 The list note should be invoked only once when you initially load the notes page, rather than each time you filter the notes by category.